public/generated.json

```diff
@@ -0,0 +1,191 @@
+[
```

Comment:
I'd remove this file from `public/`, maybe we can build something like /sandbox/data/ to store this kind of files.